.github/workflows/eaa_kbc.yml

```diff
@@ -14,45 +16,35 @@ jobs:
   eaa_kbc_ci:
     if: github.event_name == 'pull_request'
     name: Check
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         rust:
           - stable
+
+    container: runetest/compilation-testing:ubuntu18.04
           
     steps:
       - name: Code checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 1
 
-      - name: Set up docker
-        uses: docker-practice/actions-setup-docker@master
-
-      - name: Install Rust toolchain (${{ matrix.rust }})
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ matrix.rust }}
-          override: true
-          components: rustfmt
-
-      - name: Create rats-tls compiling container
+      - name: Install Rust toolchain
         run: |
-          rats_compile_env=$(docker run -itd --privileged --rm --net host -v /usr/local/lib:/usr/local/lib inclavarecontainers/test:compile-check-ubuntu18.04);
-          echo "rats_compile_env=$rats_compile_env" >> $GITHUB_ENV
+          apt-get update && apt-get install -y cargo;
+          cp -r /root/.cargo /github/home/.cargo;
+          rustup install stable;
+          rustup default stable;
```

Comment:
1. What was wrong with the rust installation via `actions-rs/tollchain` ?
2. Doesn't rustup install cargo? Do we need to install cargo via apt?
3. I think the semicolons at the end of the lines are not needed in this file (everywhere). But please double-check.

Reply:
I tried to use `actions-rs/tollchain`, but there will be some problems in the container, because the default installation path is /`root/.cargo`, and `actions-rs/tollchain` will be installed to `/home/github/.cargo`. 

`apt-get install cargo` will install rustup and cargo at the same time, which is simpler than installing rustup first and then installing cargo with rustup.